### PR TITLE
Release MU01 to Baklava

### PR DESCRIPTION
**Pull request by bowd**
### Description

This PR is a result of doing a final MU01 release on baklava and cleaning up the broadcast folder.
During the release process, a few more sidequests were accomplished in order to fix the issues that came up.

### Other changes

- Move configuration structs to a library and implement chain dependent variations. This came up when realizing that the minimum number of rates required will depend on the environment from now on.
- Better parameterize the `approve_and_execute` script.
- Write a quick `deploy` script that runs all deployment scripts in turn
- `console2` -> `console`

### Tested

It was released!

### Related issues

N/A

### Backwards compatibility

N/A

### Documentation

N/A

### CODE REVIEW

File: script/upgrades/MU01/governance/Config.sol [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+import { Contracts } from "script/utils/Contracts.sol";
+import { Arrays } from "script/utils/Arrays.sol";
+
+library Config {
+  using FixidityLib for FixidityLib.Fraction;
+  using Contracts for Contracts.Cache;
+
+  uint8 private constant L0 = 1; // 0b001 Limit0
+  uint8 private constant L1 = 2; // 0b010 Limit1
+  uint8 private constant LG = 4; // 0b100 LimitGlobal
```
Comment (tobikuhlmann) [inline script/upgrades/MU01/governance/Config.sol:19]: is 4 on purpose, not 3 like consecutive numbers?

File: bin/cgp.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Imperfect but simple script to pass governance proposals on Celo tesnets (staging, baklava or alfajores)
```
Comment (nvtaveras) [inline bin/cgp.sh:3]: update this? Looks like it was taken from the other script. Also maybe add an example usage?

File: bin/approve_and_execute.sh [hunk 3]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Imperfect but simple script to pass governance proposals on Celo tesnets (staging, baklava or alfajores)
```
Comment (nvtaveras) [inline bin/approve_and_execute.sh:3]: nit: add an example usage

File: bin/deploy.sh [hunk 4]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Imperfect but simple script to pass governance proposals on Celo tesnets (staging, baklava or alfajores)
```
Comment (nvtaveras) [inline bin/deploy.sh:3]: update this comment?

File: contracts/MockERC20.sol [hunk 5]
```diff
@@ -11,7 +11,7 @@ contract MockERC20 is ERC20, ERC20Detailed, Ownable {
     string memory symbol,
     uint8 decimals
   ) public ERC20Detailed(name, symbol, decimals) {
-    mint(msg.sender, 1000000 ether);
+    mint(msg.sender, 100_000_000 ether);
```
Comment (nvtaveras) [inline contracts/MockERC20.sol:14]: 🤑

File: script/upgrades/MU01/tests/DeploymentChecks.sol [hunk 6]
```diff
@@ -129,27 +129,25 @@ contract DeploymentChecks is Script {
   function checkReserveMultisigCanSpend() public {
     uint256 oneMillion = 1_000_000 * 1e18;
 
-    assert (address(reserve).balance == 0);
-    assert (MockERC20(bridgedUSDC).balanceOf(address(reserve)) == 0);
+    // assert (address(reserve).balance == 0);
+    // assert (MockERC20(bridgedUSDC).balanceOf(address(reserve)) == 0);
```
Comment (nvtaveras) [inline script/upgrades/MU01/tests/DeploymentChecks.sol:133]: remove?

**Review by nvtaveras — CHANGES_REQUESTED** [review]

File: script/upgrades/MU01/governance/Config.sol (hunk 1 above)
Comment (bowd) [reply script/upgrades/MU01/governance/Config.sol:19]: Yeah, these are flags, think of them as switches that can be on or off, and which can be encoded into a single number, and thus a single variable.
```
L0             = 0b001; // in binary
L1              = 0b010;
LG             = 0b100;
L0_L1        = 0b011; // L0 | L1 == L0 + L1
L0_L1_LG = 0b111;  // L0 | L1 | LG == L0 + L1 + LG
```

**Review by tobikuhlmann — APPROVED** [review]
yup, neat looking bash script descriptions :)

**Review by nvtaveras — APPROVED** [review]
🚀🚀🚀🚀 🌑 🌑 🌑 🌑

**Pull request merged**